File: Daniil-Kadochnikov/Second Term/First block/Filters/Filters/SobelXFilter.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Filters
{
	public class SobelXFilter: AKernelFilter
	{
		public new int[] Kernel = { -1, 0, 1, -2, 0, 2, -1, 0, 1 };
		public new int Divider = 1;
	}
}

File: Khelmianov-Artem/Term2/Block1/Filters/src/main/kotlin/bmp/filters/SobelXFilter.kt
package bmp.bmp.filters

import bmp.bmp.AKernelFilter

object SobelXFilter : AKernelFilter() {
    override var radius = 1
    override var kernel = listOf(
        -1f, -2f, -1f,
        0f, 0f, 0f,
        1f, 2f, 1f
    )
}